src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs

```diff
@@ -35,6 +35,14 @@ public class ServiceControlCoreTransports
                 Matches = name => name.Equals("ServiceControl.Transports.ASB.ASBForwardingTopologyTransportCustomization, ServiceControl.Transports.ASB", StringComparison.OrdinalIgnoreCase)
             },
             new TransportInfo
+            {
+                Name = "AzureServiceBus - .Net Standard",
```

Comment:
Why is there a hyphen in the name?

Reply:
Also, it should be ".NET", not ".Net"

Reply:
That name has no real relevance other than it is displayed ik the test output. This reflects whatever that has been typed in there first and can be considered a braindump

Reply:
I made a mistake here. I commented on the wrong file. Here it definitely is relevant what we write! Will change